packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -191,6 +200,7 @@ export function SynapseTable(props: SynapseTableProps) {
       columnVisibility: columnVisibility,
     },
     meta: {
+      rowSet,
```

Comment:
Pass the `rowSet` in the table meta so it can be used in the table cells as needed